### PR TITLE
Update documentation to reflect the new practices namings

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR  updates documentation to reflect the new practices namings.
Context: **RFC** https://github.com/artsy/README/issues/349

### CODE REVIEW

File: practices/mobile.md [hunk 1]
```diff
@@ -1,56 +1,56 @@
...
-  anticipate how iOS code will continue to operate in production for _years_).
+- Testing is different from other Artsy systems (Web and Platform practices test on staging; we test in betas).
+- Deploying is different from other Artsy systems (Web and Platform practices promote from staging to production;
+  for iOS, we cut betas and submit to Apple for App Store approval).
+- Long-term maintenance is different (Wen and Platform practices deploy code to hardware they control and can patch
+  easily; we deploy code to hardware our users control and can't guarantee upgrading at all, so we have to
+  anticipate how mobile code will continue to operate in production for _years_).
 
-We do have a lot in common with the [Front-End Practice](./front-end.md):
+We do have a lot in common with the [Wen Practice](./web.md):
```
Comment (admbtlr) [inline practices/mobile.md:20]:
```suggestion
We do have a lot in common with the [Web Practice](./web.md):
```

File: practices/mobile.md [hunk 2]
```diff
@@ -1,56 +1,56 @@
...
+is to support them contributing to our mobile projects (mostly via React Native, but
 [sometimes in native code](http://artsy.github.io/blog/2018/06/15/cocoapods-keys-react-native/)).
 
 ## Team Information
 
-Engineers who work on iOS code at artsy are included in the **@ios-front-enders** Slack group. They'll be
-automatically added to the #front-end-ios Slack channel. The Front-End iOS Practice meetings every two weeks;
-meeting notes are
-[available in Notion](https://www.notion.so/artsy/Front-End-iOS-ecc07763bfd04a848c74107dde3ec6dc).
+Engineers who work on mobile code at artsy are included in the **@ios-front-enders** Slack group. They'll be
```
Comment (admbtlr) [inline practices/mobile.md:32]: Hasn't the Slack group name changed?

**Review by admbtlr — APPROVED** [review]
Thanks for doing this! Just a couple of small things.

File: practices/mobile.md (hunk 2 above)
Comment (MounirDhahri) [reply practices/mobile.md:32]: no actually, it still hasn't changed. We sent an email to IT to update it but we still haven't heard back from them. I will update it once it's done

**Review by ashfurrow — APPROVED** [review]
Looks good 👍

**Review by brainbicycle — APPROVED** [review]

File: practices/mobile.md (hunk 2 above)
Comment (SamRozen) [reply practices/mobile.md:32]: @MounirDhahri I can update it, what do you want to change it to?

Comment (MounirDhahri) [reply practices/mobile.md:32]: Thanks. we would like to update it to `@mobile-practice`

Comment (MounirDhahri) [reply practices/mobile.md]: I updated the PR to include the new group slack name, please feel free to merge it once you update it 🙏

Comment (SamRozen) [reply practices/mobile.md]: done

**Pull request merged**